src/modules/about.js

```diff
@@ -0,0 +1,46 @@
+const mother = document.getElementById('content');
+
+function makeAboutCard(title, text) {
+  const aboutCardBody = document.createElement('div');
+  aboutCardBody.className = 'col-12';
+
+  const aboutCardTitle = document.createElement('h2');
+  aboutCardTitle.className = 'fw-bold  my-4 text-end text-warning text-warning';
+  const aboutCardTitleText = document.createTextNode(title);
+  aboutCardTitle.appendChild(aboutCardTitleText);
+
+  aboutCardBody.appendChild(aboutCardTitle);
+
+  const aboutCardText = document.createElement('p');
+  aboutCardText.className = 'card-text fw-bold text-end text-warning';
+  const aboutCardTextText = document.createTextNode(text);
+  aboutCardText.appendChild(aboutCardTextText);
+
+  aboutCardBody.appendChild(aboutCardText);
+  return aboutCardBody;
+}
```

Comment:
:bulb: Make use of arrow functions instead of the old `function` keyword when declaring your functions. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

Reply:
Done!